src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyService.kt

```diff
@@ -116,6 +120,62 @@ class NotifyAppointmentService(
         event.actionPlanSession.sessionNumber,
       )
 
+      when (event.type) {
+        ActionPlanAppointmentEventType.ATTENDANCE_RECORDED -> {
```

Comment:
🙋‍♂️ Could this be a code smell? This `when` block is identical to the one below. Do we need the `ActionPlanAppointmentEvent`?